src/rpdk/core/data/managed-upload-infrastructure.yaml

```diff
@@ -38,6 +38,24 @@ Resources:
               - !Sub "arn:${AWS::Partition}:s3:::${ArtifactBucket}"
               - !Sub "arn:${AWS::Partition}:s3:::${ArtifactBucket}/*"
 
+  ArtificatBucketSecureCopyPolicy:
+    Type: AWS::S3::BucketPolicy
+    Properties:
+      Bucket: !Ref ArtifactBucket
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
```

Comment:
This could also be added to the above bucket policy as another statement

Reply:
Folded "secure transport" policy into ArtifactCopyPolicy.